node/pkg/watchers/ibc/watcher_test.go

```diff
@@ -213,3 +214,39 @@ func TestParseIbcAllChannelChainsQueryResults(t *testing.T) {
 	assert.Equal(t, expectedChannStr2, result.Data.ChannelChains[1][0].(string))   //nolint:forcetypeassert
 	assert.Equal(t, uint16(22), uint16(result.Data.ChannelChains[1][1].(float64))) //nolint:forcetypeassert
 }
+
+func TestConvertingWsUrlToHttpUrl(t *testing.T) {
+	assert.Equal(t, "http://wormchain:26657", convertWsUrlToHttpUrl("ws://wormchain:26657/websocket"))
+	assert.Equal(t, "http://wormchain:26657", convertWsUrlToHttpUrl("ws://wormchain:26657"))
+	assert.Equal(t, "http://wormchain:26657", convertWsUrlToHttpUrl("wormchain:26657"))
+}
+
+func TestParseAbciInfoResults(t *testing.T) {
+	// This came from the following query: http://localhost:26659/abci_info
+	respJson := []byte(`
+{
+  "jsonrpc": "2.0",
+  "id": -1,
```

Comment:
It's okay to use a negative number?  That works?
Learn something new everyday...